### ISSUE TITLE
Code: VPN Resource center

### ISSUE THREAD

**Issue by slightlyoffbeat** [mozilla/bedrock#10504]
## Description

Early designs are done for the VPN knowledge center. It would be best to jump into code early and start building this.

This may eventually be powered by Contentful, however for this task, we will build in bedrock. We can then work backwards and see how to best port this to contentful.

While developing, please take note of what elements are in protocol and what elements are not.

The purpose here is to get into code early. Designs and copy are flexible and will change as we work this out.

---

## Before starting:

- Reach out to Elise for Designs and
- Reach out to Dan Brown for a quick overview of vision for this page.

## Scope
We are building an MVP. The scope of this MVP includes:
- Knowledge Center landing page
- Knowledge Center individual pages

Not in scope (for this issue)
- Navigation changes
- Categories.

## Resources:
- [Designs](https://www.figma.com/file/xo5JhiQIlpGWkb9lgh0ccZ/VPN-Resource-Guide?node-id=0%3A1)
- Copy: Ipsum for now

#### Success Criteria
- [ ] Approval from Elise
- [ ] Creative Approval
- [ ] List of custom elements that need to be either 1. added to protocol or 2. added to a custom layout
- [ ] Ready to bring to Contentful
